src/utils/polygon.cpp

```diff
@@ -190,6 +190,22 @@ unsigned int Polygons::findInside(Point p, bool border_result)
     return ret;
 }
 
+int64_t Polygons::polyLineLength() const
+{
+    int64_t length = 0;
```

Comment:
It makes sense to make this coord_t, but coord_t is long long and I don't know whether this is int64_t rather than coord_t for a reason....